src/components/v2/V2Create/forms/RulesForm/index.tsx

```diff
@@ -14,6 +14,8 @@ import ReconfigurationStrategySelector from 'components/shared/ReconfigurationSt
 
 import { BallotStrategy } from 'models/ballot'
 
+import _ from 'lodash'
```

Comment:
nit: importing the entire lodash lib is going to import the library when we only need one or two functions.

https://www.npmjs.com/package/lodash.isequal

https://lodash.com (it's also in the docs)

```
// Cherry-pick methods for smaller browserify/rollup/webpack bundles.
var at = require('lodash/at');
var curryN = require('lodash/fp/curryN');
```

Reply:
Good catch - thanks @benschac!